buttplug/src/device/protocol/lovense.rs

```diff
@@ -57,6 +61,10 @@ impl ButtplugProtocol for Lovense {
     device_impl: &DeviceImpl,
   ) -> BoxFuture<'static, Result<Option<String>, ButtplugError>> {
     let subscribe_fut = device_impl.subscribe(DeviceSubscribeCmd::new(Endpoint::Rx));
+
+    // Need the 150ms delay here for some devices
+    thread::sleep(Duration::from_millis(150));
```

Comment:
I wanted to use `Delay::new()` to be consistent, but I couldn't use await in this context and I'm still not sure if Box::pin serially with the calling code.

Reply:
thread sleep blocks, this would both block all execution and not compile on WASM at all. I'll take care of it from here.